lib/arsi/relation.rb

```diff
@@ -1,6 +1,6 @@
 module Arsi
```

Comment:
⚠️ I'm seeing this warning when running specs:

> [/lib/arsi/relation.rb:17](https://github.com/zendesk/arsi/blob/c7a61cc9776ddff71c6215681985e2ccbcb4d2bd/lib/arsi/relation.rb#L17): warning: instance variable @without_arsi not initialized

Reply:
Yeah. I'm not sure the best way. I looked into that but we'd need to tie into the initialization of the object so that the variable exists when it's created. I'll give it another go and see if I can think of something

Reply:
Would something like this make sense?
```ruby
    def without_arsi?
      defined?(@without_arsi) || !arsi_scopeable?
    end
```

Reply:
This is actually a bug. It should be:

```
    def without_arsi?
      return @without_arsi if defined?(@without_arsi) && @without_arsi
      !arsi_scopeable?
    end
```